### PR TITLE
Cleanup grammars and bump to 0.2.4 release

### PR DESCRIPTION
**Pull request by ghost**
* Reorder into alphabetical order based on the meson git repository (https://github.com/mesonbuild/meson/blob/master/data/syntax-highlighting/vim/syntax/meson.vim).

* Reorder into alphabetical order based on the official website (https://mesonbuild.com/Reference-manual.html#builtin-objects).

* Add `in` as an operator (https://github.com/mesonbuild/meson/pull/6776), (https://github.com/mesonbuild/meson/blob/master/test%20cases/common/17%20comparison/meson.build).

* Add `continue` and `break` statements (https://github.com/mesonbuild/meson/pull/7478), (https://mesonbuild.com/Syntax.html#foreach-break-and-continue).

* Add `in` and `not in` operators in test file.

* Bump release to 0.2.4, since the previous changes it has not been updated.

### CODE REVIEW

**Comment by ghost** [comment]
**Note**: The community package is still on the version 0.2.1 (https://atom.io/packages/language-meson).

File: grammars/meson.json [hunk 1]
```diff
@@ -16,11 +16,11 @@
...
     },
     {
-      "match": "\\b(if|else|elif|endif|foreach|endforeach)\\b",
+      "match": "\\b(elif|else|if|endif|foreach|endforeach|continue|break)\\b",
       "name": "keyword.control.flow.meson"
     },
     {
-      "match": "\\b(and|not|or)\\b",
+      "match": "\\b(and|not|or|in)\\b",
       "name": "keyword.operator.logical.meson"
```
Comment (ghost) [inline grammars/meson.json:24]: @TingPing Maybe you would want to rename this. (as `in` is not a logical operator)

Comment (TingPing) [reply grammars/meson.json:24]: The names aren't defined by me, they are part of themes. Feel free to find another more accurate one.

**Comment by TingPing** [comment]
> Note: The community package is still on the version 0.2.1 (atom.io/packages/language-meson).

I don't use atom personally anymore, I keep this around for github theming tbh. *maybe* I can add a new maintainer or somebody else can manage uploading it?

**Comment by ghost** [comment]
@TingPing Sure I would love to do it, as I use meson everyday :).

**Comment by TingPing** [comment]
I've invited you to the repo. I'm not sure how `apm` publishing handles permissions but maybe it will just work. Let me know :)

(Please still have changes go through PRs, this is just for publishing releases)

**Comment by ghost** [comment]
@TingPing I will check this tonight and will let you know.

And of course I am only doing changes through PRs first, as in every open-source project :).

PS: Should I make 0.2.5 on GitHub after the latest changes?

**Comment by TingPing** [comment]
> PS: Should I make 0.2.5 on GitHub after the latest changes?

I'd just tag 0.2.4 on master.

File: grammars/meson.json (hunk 1 above)
Comment (ghost) [reply grammars/meson.json:24]: After review this is the correct identifier.

**Comment by ghost** [comment]
@TingPing Successfully updated via apm.

**Pull request merged**